src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: AWS System Manager Distributor
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's AWS System Manager Distributor monitoring integration: what data it reports, and how to enable it."
```

Comment:
actually AWS System Manager Distributor is a way to provide our infrastructure agent to new and old customers. It's a way of easier installation in a diverse environment. It's very different from AppRunner integration. We do not make any monitoring of SMD, no any extra data, no way to enable it. The point of sale here is to install our infrastructure agent on a fleet of different in OS and architecture instances by a single click. Monitoring goes only for the instance, the agent is installed on and by the way the infra agent does so, noth
ing added, nothing removed

Reply:
Ah interesting! I just took another pass to clarify this